package.json

```diff
@@ -125,18 +125,14 @@
     "build:types": "tsc",
     "watch": "babel --watch src/ -d lib/ --copy-files",
     "watch:ts": "tsc --watch",
-    "test:mongodb:runnerstart": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=$npm_config_dbversion} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} mongodb-runner start -t ${MONGODB_TOPOLOGY} --version ${MONGODB_VERSION} -- --port 27017",
```

Comment:
It seems this removes the separation between starting the mongodb server and running the tests with `testonly`. Starting the mongodb server takes some time, which is inefficient and introduces a lag when frequently running tests during development. This can currently be avoided by starting the mongodb server once and running `testonly` when needed.

Reply:
Not sure to understand @mtrezza because the currently the pretest and post hook run always when using "npm test xxxxx"

I always used the "npm test" script when contributing , and the mongo startup time was not really an issue.

But we can restore a start/stop mongodb command, i'm just afraid of a developer using start/stop mongo db and "npm test" leading to start up issue.

I can restore mongo db start/stop if you want and to support both testing pattern :)